### ISSUE TITLE
Error after reload on Expo Dev Client

### ISSUE THREAD

**Issue by Amrit0991** [OP-Engineering/op-sqlite#21]
Getting the following error on Expo Dev Client after reloading:

```
Error: Failed to install op-sqlite, the native initializer function does not exist. Are you trying to use OPSQLite from different JS Runtimes?, js engine: hermes
```

This issue has already been fixed in another library (react-native-mmkv) with the following PR:

https://github.com/mrousavy/react-native-mmkv/pull/602/files

Happy to make a PR for this library, or if you prefer to take care of it yourself, totally fine too.

**Comment by ospfranco** [comment]
Nice, yeah, please create a PR and test it on a non-expo app too please

**Comment by ospfranco** [comment]
I tested the change, everything seems to be working. Released a new version `2.0.1`

**Comment by Amrit0991** [comment]
Appreciate you!
